tests/edgecluster/data/missions/deployment-to-all.yaml

```diff
@@ -3,7 +3,7 @@ kind: Mission
 metadata:
   name: deployment-to-all
 spec:
-  content: |
+  missionresource: |
```

Comment:
come to think, "content" is a big general but okay. any other reason it's changed to "missionresource"?